src/executables/util_cs.c

```diff
@@ -3296,17 +3328,41 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
 				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
 				     &applied_final_lsa);
+	  if (error != NO_ERROR)
+	    {
+	      goto print_applyinfo_usage;
+	    }
 	}
 
     check_applied_info_end:
       if (error != NO_ERROR)
 	{
 	  PRINT_AND_LOG_ERR_MSG ("%s\n", db_error_string (3));
+	  goto print_applyinfo_usage;
 	}
       error = NO_ERROR;
 
       if (check_master_info)
 	{
+	  check_master_info = false;
+	  for (; ha_node_list_pp[ha_node_cnt] != NULL;)
+	    {
+	      if (!strcmp (master_node_name, ha_node_list_pp[ha_node_cnt]))
+		{
+		  check_master_info = true;
+		  break;
+		}
+	      ha_node_cnt++;
+	    }
+	  if (!check_master_info)
+	    {
+	      er_init (er_msg_file, ER_NEVER_EXIT);
+	      error = INVALID_SOCKET;
+	      er_set_with_oserror (ER_ERROR_SEVERITY, ARG_FILE_LINE, ERR_CSS_TCP_HOST_NAME_ERROR, 1, master_node_name);
+	      PRINT_AND_LOG_ERR_MSG ("\n%s\n\n", db_error_string (3));
```

Comment:
1. Do we have to set an error here? and INVALID_SOCKET seems never used.
2. Why is `error` set to as INVALID_SOCKET, and do you use another for `er_set_with_oserror()`

Reply:
1. I verify `INVALID_SOCKET` is not necessary to be set here. I will remove it.
2. This is the translating code from `css_sockaddr()` when `gethostbyname()` is failed. But now I knew that is not necessary.